ntc_templates/templates/alcatel_aos_show_mac-address-table.textfsm

```diff
@@ -0,0 +1,18 @@
+Value VLAN_ID (\d+)
+Value DOMAIN (\S+)
+Value MAC (\S+)
+Value TYPE (\S+)
+Value PROTOCOL (\S+)
+Value OPERATION (\S+)
+Value INTERFACE (\S+)
+
+Start
+  ^\s*Vlan\s*Mac\s*Address\s*Type\s*Protocol\s*Operation\s*Interface\s*$$
+  ^\s*------+-------------------+--------------+-----------+------------+-----------\s*$$
+  ^\s*${VLAN_ID}\s*${MAC}\s*${TYPE}\s*(---|${PROTOCOL})\s*${OPERATION}\s*${INTERFACE}\s*$$ -> Record
+  ^\s*Domain\s*Vlan\/SrvcId\[ISId\/vnId\]\s*Mac\s*Address\s*Type\s*Operation\s*Interface\s*$$
+  ^\s*------------+----------------------+-------------------+------------------+-------------+-------------------------\s*$$
```

Comment:
```suggestion
  ^\s*-+\+
  ^\s*${VLAN_ID}\s*${MAC}\s*${TYPE}\s*(---|${PROTOCOL})\s*${OPERATION}\s*${INTERFACE}\s*$$ -> Record
  ^\s*Domain\s*Vlan\/SrvcId\[ISId\/vnId\]\s*Mac\s*Address\s*Type\s*Operation\s*Interface\s*$$
```

I feel comfortable that we are only matching table dividers with looking for lines that have `-` repeated followed by a `+`